src/js/components/Notification/Notification.js

```diff
@@ -20,10 +20,10 @@ const Notification = ({ message, onClose, status, title, toast }) => {
   }, [onClose]);
 
   useEffect(() => {
-    const timer = setTimeout(close, theme.notification.time);
+    const timer = setTimeout(close, theme.notification.toast.time);
```

Comment:
This would be a breaking change. I think we need to change it to be backwards compatible.
```suggestion
    const timer = setTimeout(close, theme.notification.toast.time || theme.notification.time);
```